File: src/endpoints.ts
export enum Endpoints {
	// for dev purpose
	create_account = "/dev/createaccount",
	verify_login = "/dev/authenticate",
	verify_cookie = "/dev/validate_cookie",
	insert_gita_shloka = "/dev/addgitashloka",
	insert_tandava_shloka = "/dev/addtandavashloka",

	// redirects
	docs = "/docs",
	discord = "/discord",
	github = "/github",

	// exposed to user
	home = "/",

	bhagavadgita = "/api/v1/bhagavadgita/:adhyaya/:shloka",
	bhagavadgita_query = "/api/v1/bhagavadgita/query",
	ramcharitmanas = "/api/v1/ramcharitmanas/:kanda/:shloka",
}

File: src/endpoints.py
class Endpoints:
    _INDEX = "/"

    # developer endpoints 
    SLASHN_GENERATOR = "/_dev/slashn/{text}"

    # bhagavadgita shlokas
    BHAGAVADGITA_SHLOKA = "/bhagavadgita/{adhyaya}/{shloka}"
    BHAGAVADGITA_QUERY = "/bhagavadgita/query"

    # shiv tandava strotam
    SHIV_TANDAVA = "/shivtandava"
    SHIV_TANDAVA_SHLOKA = "/shivtandava/{shloka}"


endp = Endpoints()
